R/transform-code.R

```diff
@@ -110,6 +170,10 @@ get_engine_pattern <- function() {
 #'
 #' @inheritParams separate_chunks
 #' @keywords internal
-get_knitr_pattern <- function(lines) {
-  knitr::all_patterns[["md"]]
+get_knitr_pattern <- function(lines, filetype = "Rmd") {
+  if(filetype == "Rnw") {
+    knitr::all_patterns[["rnw"]]
+  } else {
```

Comment:
Here also I'd be explicit on the `else` part

Reply:
👍